modules/odc-customizing-available-cluster-roles-using-developer-perspective.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// applications/projects/working-with-projects.adoc
+
+[id="odc-customizing-available-cluster-roles-using-developer-perspective_{context}"]
+= Customizing the available cluster roles using the Developer perspective
+
+You can customize cluster role's name of existing projects by editing the YAML code in the `console.openshift.io/display-name: v1`.
+
+.Procedure
+Customize the cluster role options of all projects that are shown in the *Project Access* tab:
+
+. In the *Search* view, select the *Resources* drop down and search "*Console*".
```

Comment:
Your previous sentence says that we want to be in the *Project Access* tab. In your first step, you tell me to go to the *Search* view -> *Resources* -> *Console* So am I supposed to be in the Console or Project Access tab? Or is the Console within the Project Access tab?

Honestly, I'm not even sure if you need the first sentence before the procedure, especially since it can cause confusion as to where to direct the user. I think if you make the use case clearer in the abstract, you can simply start with step 1 under the procedure. So: 

**Title**
<abstract>
**Procedure**
1....

Reply:
Sounds good. Will try to simplify it a bit more.